website/docs/advanced-concepts/version-control-with-git/guides/resolve-merge-conflicts-in-git.md

```diff
@@ -0,0 +1,123 @@
+# Resolve Merge Conflicts in Git
+
+Merge conflicts are a regular part of working with Git, arising when changes from separate branches cannot be merged. 
+
+This guide provides strategies for resolving different types of merge conflicts, ensuring smoother collaboration and project management in Git.
+
+
+
```

Comment:
We should write the 2 cases when merge conflicts arise

1. When trying to merge your changes to the common branch
2. When the remote of the branch you're working on has been updated and you make changes to the same pages as the update in your local. In this case, you should explain the exact case, how it looks in Appsmith commit modal and how they need to create a new branch etc.